src/Codeception/Step/Comment.php

```diff
@@ -1,37 +1,41 @@
 <?php
+
+declare(strict_types=1);
+
 namespace Codeception\Step;
 
 use Codeception\Lib\ModuleContainer;
 use Codeception\Step as CodeceptionStep;
+use function mb_strcut;
 
 class Comment extends CodeceptionStep
 {
-    public function __toString()
+    public function __toString(): string
     {
         return (string) $this->getAction();
     }
 
-    public function toString($maxLength)
+    public function toString($maxLength): string
     {
-        return mb_strcut($this->__toString(), 0, $maxLength, 'utf-8');
+        return mb_strcut((string) $this, 0, $maxLength, 'utf-8');
     }
 
-    public function getHtml($highlightColor = '#732E81')
+    public function getHtml($highlightColor = '#732E81'): string
     {
         return '<strong>' . $this->getAction() . '</strong>';
     }
 
-    public function getPhpCode($maxLength)
+    public function getPhpCode($maxLength): string
```

Comment:
$maxLength should be int, right?

Reply:
Yes, what happens is that to add this type-hint I should also change the abstract class as well, and I didn't decide to touch those files yet (`src/Codeception/*.php`). I'll do it here anyway.

Reply:
I will make this change in a future PR when I edit the files in the path: `src/Codeception/*.php`